app/static/js/pico_recipe.js

```diff
@@ -122,13 +131,17 @@ var recipe_table = {
         {
             formatter: plusIcon, width: 49, hozAlign: "center",
             cellClick: function (e, cell) {
-                cell.getTable().addRow({}, false, cell.getRow());
+                if (rowIsEditable(cell))
+                    cell.getTable().addRow(Object.assign({}, cell.getRow().getData()), false, cell.getRow()).then(function(row) {
+                        row.update({name: "New Step"});

```

Comment:
the only outstanding issue I see from my testing tonight is there is another situation in the hop/boil time calculation that isn't taking into account multiple steps for one adjunct so the new hop/boil time calculation upon creating a new row is off (but that would be the case before this as well... I didn't take that into account).

Reply:
though I'd imagine that is fairly rare, maybe getting there with the last addition for DME pauses... so I'll look at reworking that a bit later, not going to say this is blocked on it just appears weird is all (though I kinda liked that the row was invalid upon a copy requiring the user to add the data explicitly 🤷‍♂️ )